packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -129,6 +130,17 @@ class BuildInfo {
   /// rerun tasks.
   final String? performanceMeasurementFile;
 
+  /// Configure a constant pool file.
+  /// Additional constant values to be made available in the Dart program.
+  ///
+  /// These values can be used with the const `fromEnvironment` constructors of
+  ///  [String],the key is json field,the value is
```

Comment:
nit: `/// [String],the key and field are json values`
Also delete line 141

Reply:
OK, my translation is still unclear